subjects/weareunique/README.md

```diff
@@ -0,0 +1,57 @@
+## weareunique
+
+### Instructions
+
+Write a function that takes two strings and returns the number of characters that are not included in both, without doubles.
+- If there is no unique characters return `0`.
+- If both strings are empty return `-1`.
+
+### Expected function
+
+```go
+func WeAreUnique(str1 , str2 string) int {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
```

Comment:
-  'func main' needs to be formatted correctly in order to match the subjects structure like we have been doing in the previous exercises.

Reply:
Agreed. Fixed